### DISCUSSION TITLE
The specified file can not be found. Could not load file or assembly 'Macad.Managed'

### DISCUSSION THREAD

**Discussion by victcdva** [Macad3D/Macad3D#21] [Q&A]
Sup, 
System.IO.FileNotFoundException : 'Could not load file or assembly 'Macad.Managed, Version=2.0.1436.0, Culture=neutral, PublicKeyToken=f3b86e0df7940e9a'. The specified file can not be found.'

![image](https://user-images.githubusercontent.com/38483934/167054893-a0233952-5c65-4096-a763-b22a7c070e70.png)

I get this error when I try to execute the project in Debug, I guess this error is similar to
[https://github.com/Macad3D/Macad3D/discussions/17](url)

but this not resolve it for me

am I missing a library? I followed the repository documentation,

I built in Debug and in Release

`>build all`

in Release the project compiles fine

I get this when I try compile in Debug

Thanks in Advance

**Comment by dipts** [comment]
Hi,
the process is started in 32bit environment and thus can not load the mixed mode assembly.
Please select "x64" for platform, instead of "AnyCpu".

**Reply by victcdva** [reply]
@dipts  thank you for the tip, but I still have the exception, I followed the documentation but I still have the error, I don't understand what happen

**Reply by dipts** [reply]
Then there remains the possibility that a dependency is missing. Can you check with a tool like [Dependency Walker](https://www.dependencywalker.com/) if all dependent DLLs are found?
